src/qt/consolidateunspentdialog.h

```diff
@@ -3,7 +3,9 @@
 
 #include <QDialogButtonBox>
 #include <QDialog>
+#include <QWizard>
 #include <QString>
+#include <QLabel>
```

Comment:
```suggestion
#include <QString>
```

Doesn't look like these ended up being used for the original consolidate dialog.